api/handlers/tag_handler_test.go

```diff
@@ -173,7 +182,51 @@ func TestFindTags_byVersion(t *testing.T) {
 				read(&tags)
 
 				r.AssertEqual(len(tags), 1)
-				r.AssertEqual(tags[0].EntityID, "d2")
+				r.AssertEqual(tags[0].EntityID, "d.2")
+			},
+		},
+		{
+			Name: "7",
+			Request: &TestRequest{
+				Query: "version=7",
+			},
+			Run: func(r *testutils.Reporter, _ interface{}, req *restful.Request, resp *restful.Response, read Readf) {
```

Comment:
I've added the `callAndRead` function to remove some of the duplicate code in these cases.